app/build.gradle

```diff
@@ -353,7 +353,7 @@ distributions {
                 exclude 'es.jar'
                 exclude 'es-*.jar'
             }
-            into("lib") {
+            into("lib/main") {
```

Comment:
this could potentially break packaging. we need to make sure distribution builds work thereafter!